docs/source/graph-construction/gs-processing/gspartition/sagemaker.rst

```diff
@@ -0,0 +1,134 @@
+===================================
+Running partition jobs on SageMaker
+===================================
+
+Once the :ref:`distributed processing setup<gsprocessing_distributed_setup>` is complete,
+you can use Amazon SageMaker launch scripts to launch distributed processing jobs with AWS resources.
+
+Build the docker image for partition jobs on SageMaker
+------------------------------------------------------
+GSPartition job on SageMaker uses SageMaker's **BYOC** (Bring Your Own Container) mode.
+
+Step 1: Build a SageMaker-compatible Docker image
+...................................................
+
+.. note::
+    * Please make sure your account has access key (AK) and security access key (SK) configured to authenticate accesses to AWS services.
```

Comment:
We can link to an example policy here? https://docs.aws.amazon.com/AmazonECR/latest/userguide/security_iam_id-based-policy-examples.html#security_iam_id-based-policy-examples-access-one-bucket